### ISSUE TITLE
Update for Fund6

### ISSUE THREAD

**Issue by coire1** [Project-Catalyst/vca-tool#11]
Here is the recap of the updates that should be done for Fund6.
It probably makes sense to split this issue into many, or define specific tasks.

The main update is related to the "CSV" format imported and export that will be different in this fund.
Here is the link to the new CSV Format: https://gist.github.com/coire1/73cf7cd6ffc323d7daf9c6b282168faf
- The assessment is not anymore for a single criteria, but it includes the whole triplet (3 criteria), so 3 notes and 3 ratings. In addition to "proposer mark" is now visible also the "Proposer Not Valid rationale"
- The criteria that could be selected in by a vCA are only 3: "excellent", "good", "not valid". The long list of the old criteria is now more limited, and the "rationale" for vCAs is not present anymore.
    
So basically the changes to be implemented:
- `Assessment Preview` should be updated according to the new data model (3 notes, 3 ratings, Proposer Not Valid rationale visible etc.)
- `CFilters` should be updated according to the new data model (rating is not singular anymore, the note length is cumulative within the 3 notes)
- `Assessment` view should be updated according to the new data model and with the new criteria used ("excellent", "good", "not valid")
- The export mechanism should be updated to new data model

**Comment by tstachl** [comment]
New format columns:

- `id` => Number
- `Idea Title` => String
- `Idea URL` => String
- `Assessor` => String
- `triplet_id` => String
- `proposal_id` => Number
- `Impact / Alignment Note` => String
- `Impact / Alignment Rating` => Number (1-5)
- `Feasibility Note` => String
- `Feasibility Rating` => Number (1-5)
- `Auditability Note` => String
- `Auditability Rating` => Number (1-5)
- `Proposer Mark` => String (Should this just be a boolean?)
- `Proposer Not Valid rationale` => String
- `Excellent` => Unknown
- `Good` => Unknown
- `Not Valid` => Unknown

@coire1 can you double-check this, let me know if the proposer mark is a boolean, and what data type the last three columns are?

**Comment by coire1** [comment]
Yes, everything looks correct.

- `Proposer Mark` -> Boolean
- `Excellent` -> Boolean
- `Good` -> Boolean
- `Not Valid` -> Boolean

Consider that the CSV that will be imported will have an `x` string for the boolean values and when exported will transform the boolean in `x` if true and in empty string when false.
Sounds a bit strange but I have to maintain compatibility with the process that can happen in parallel in GSheet.
